dotnet/targets/Xamarin.Shared.Sdk.targets

```diff
@@ -1727,6 +1727,18 @@
 				            "
 			/>
 
+			<!-- There's no need to ship any *.o files shipped by NativeAOT -->
+			<!-- We need to remove these from ResolvedFileToPublish before calling ComputeBundleLocation, because otherwise we get warnings about no knowing what to do with these files -->
```

Comment:
nit: typo in the comment
```suggestion
			<!-- We need to remove these from ResolvedFileToPublish before calling ComputeBundleLocation, because otherwise we get warnings about not knowing what to do with these files -->
```